OrbitGl/TimeGraph.h

```diff
@@ -106,8 +107,11 @@ class TimeGraph {
   [[nodiscard]] StringManager* GetStringManager() { return string_manager_.get(); }
   void SetCanvas(GlCanvas* canvas);
   [[nodiscard]] GlCanvas* GetCanvas() { return canvas_; }
-  void SetFontSize(int font_size);
-  [[nodiscard]] int GetFontSize() { return GetTextRenderer()->GetFontSize(); }
+  void SetFontSize(uint32_t font_size);
+  [[nodiscard]] uint32_t GetFontSize() const {
+    return lround((GParams.font_size) * layout_.GetScale());
+  }
+  [[nodiscard]] uint32_t GetOverlayFontSize() const { return GParams.font_size; }
```

Comment:
I think the change in `GetFontSize` and this additional getter is misleading. If I have a getter and a setter, I expect the getter to retrieve the value that I set (maybe clamped or adjusted to fit some constraints or the like), not a new value that is calculated from the existing one. Currently, if I call `time_graph_->SetFontSize(time_graph_->GetFontSize())` 10 times when zoomed out, the font size would constantly shrink.

I would keep the existing `GetFontSize` method as it is and introduce a new `CalculateZoomedFontSize` or similar that does the job.

Reply:
ok. Sounds good! Thanks for that!